nestle.py

```diff
@@ -583,6 +651,13 @@ def update(self, pointvol):
         self.ell = bounding_ellipsoid(self.points, pointvol=pointvol)
         self.ell.scale_to_vol(pointvol)
 
+    def propose_point(self, u):
+        while True:
+            new_u = u + scale * self.ell.randoffset(rstate=self.rstate)
```

Comment:
Ah, I see. Why even separate out `propose_point()` for `ClassicSampler` then? can it just be inlined back into `ClassicSampler.new_point()` for the purpose of this PR?

(It's a bit weird to have all the queue stuff in the `Sampler.__init__` when `ClassicSampler` doesn't use it. It might be cleaner to have a derived class `EllipsoidSampler` that implements the queue stuff and then have `SingleEllipsoidSampler` and `MultiEllipsoidSampler` derive from that. But, not necessary for this PR. Just get the tests to pass. :wink: )